### ISSUE TITLE
not work

### ISSUE THREAD

**Issue by dchan1407** [Rezpector888/bot-auto-claim-freelite_coin_bot#1]
![IMG_20201025_183732](https://user-images.githubusercontent.com/72922800/97105958-47a72d80-16f1-11eb-92a8-d968c302d1fa.jpg)

can you solved this problem?

**Comment by Rezpector888** [comment]
remove the session number in folder session, then try again

**Comment by dchan1407** [comment]
yup i had done with it.... anyway a bot is scam, they not pay me 

![IMG_20201027_152906](https://user-images.githubusercontent.com/72922800/97276342-a2fc2b80-1869-11eb-9ba2-622583998dac.jpg)

**Comment by Rezpector888** [comment]
Yups they scam T_T

**Comment by dchan1407** [comment]
we hope u create a new one... may i got ur WhatsApp number to inform u if i find a legit bot/app.

thanks
